homeassistant/components/fronius/__init__.py

```diff
@@ -26,6 +34,7 @@
     FroniusPowerFlowUpdateCoordinator,
     FroniusStorageUpdateCoordinator,
 )
+from .sensor import InverterSensor
```

Comment:
We shouldn't import entities from platforms. Those are platform details and should be kept in the platforms. We can eg use our dispatch helper to signal the platform to add new entities. See eg the zwave_js integration or the tuya integration.

https://github.com/home-assistant/core/blob/4a6247f922a41f3145a30b1e071aa92f1dcdd1f1/homeassistant/components/tuya/__init__.py#L268

https://github.com/home-assistant/core/blob/4a6247f922a41f3145a30b1e071aa92f1dcdd1f1/homeassistant/components/tuya/sensor.py#L1056-L1058

Reply:
Thank you for the input. Had a look into it and this is a more elegant solution. Adopted it in PR #96782.